network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -48,11 +48,7 @@ <h1 class="h1-heading">{{ page.title }}</h1>
 {% endblock %}
 
 {% block extra_scripts %}
-  {% if coral_talk %}
-    <script src="{{ coral_talk_server_url }}static/embed.js" async onload="
-      Coral.Talk.render(document.getElementById('coral-talk-stream'), {
-        talk: '{{ coral_talk_server_url }}'
-      });
-    "></script>
+  {% if use_commento %}
+    <script async defer src="https://cdn.commento.io/js/commento.js"></script>
```

Comment:
Do we need both `async defer`? `defer` would be ignored if both are presented I _think_?